.github/workflows/maven.yml

```diff
@@ -28,5 +28,4 @@ jobs:
     - name: Test with Maven
       run: ./mvnw package -Pci-test
     - name: Codecov
-      uses: codecov/codecov-action@v1
```

Comment:
see [why-should-text-files-end-with-a-newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
woc，I heard about it for the first time